### DISCUSSION TITLE
Decimal time

### DISCUSSION THREAD

**Discussion by abonhote** [mcndt/obsidian-toggl-integration#70] [Feature Requests]
I'd love to see the time in decimal notation, so instead of 2:30 rather 2.5. It's possible with Toggl's native reports already.

**Comment by mcndt** [comment]
Hi abonhote, sorry for the late reply.

I'll create a feature request issue out of this thread so that I can track it when I have another development sprint for the plugin!

**Reply by abonhote** [reply]
Thanks, and no worries. It's working perfectly fine as it is currently.
